imjoy/__main__.py

```diff
@@ -1,12 +1,22 @@
 """Provide main entrypoint."""
-import sys
+import json
 import os
 import subprocess
-import json
+import sys
+
+from imjoy.setup import parse_cmd_line
 
 
 def main():
```

Comment:
How about rename this to other names such as `start`, to avoid the same as the one we are importing from imjoyPluginEngine.

Reply:
Yes, but I'd prefer to do that in another PR. There  are lots of lint issues left to fix. My plan is to go over one module at a time, and or one lint issue type at a time, after the refactor PR is merged.

Reply:
Ok, go ahead.